polybius.py

```diff
@@ -0,0 +1,20 @@
+
+
+SQUARE = [
+        ['e', '2', 'r', 'f', 'z', 'm'],
+        ['y', 'h', '3', '0', 'b', '7'],
+        ['o', 'q', 'a', 'n', 'u', 'k'],
+        ['p', 'x', 'j', '4', 'v', 'w'],
+        ['d', '1', '8', 'g', 'c', '6'],
+        ['9', 'i', 's', '5', 't', 'l']
+        ]
+
+def polybius(numbers):
+    # takes a string of numbers and runs them through the polybius square
+    ret = ''
+    for i in range(len(numbers))[::2]:
```

Comment:
So will we be passing in numbers comma separated? like "5,4,3,2,1"

Reply:
Sorry I probably should have been more descriptive. We can just pass "54321". Python can just index strings like an array.

Reply:
I figured that would be easier since we would probably take arguments from the command line.